key.core/src/main/java/de/uka/ilkd/key/logic/op/LocationVariable.java

```diff
@@ -30,6 +30,10 @@ public LocationVariable(ProgramElementName name, KeYJavaType t) {
         super(name, t.getSort(), t, null, false, false, false);
     }
 
+    public LocationVariable(ProgramElementName name, KeYJavaType t, Object origin) {
```

Comment:
I am not a friend of using an type `Object` here. Is there any reason why this is needed?

Reply:
I used `Object` to keep the interface flexible. I can change it to TacletApp.